cypress/cypress/e2e/AE_Tests.cy.js

```diff
@@ -357,5 +357,35 @@ describe("AE Tests", () => {
     //     expect(text).to.eq('Thank you for your review.');
     // })
   });
-  
+
+
+  it("AE_TestCase22", () => {
+        cy.visit('http://automationexercise.com');
+        cy.scrollTo('bottom')
+
+        cy.get('.recommended_items h2.title')
```

Comment:
At the last lesson, we decided to use single-line commands. Earlier, I had already fixed all the commands that were written in multiple lines.